app/assets/javascripts/admin/addon/controllers/admin-site-text-edit.js

```diff
@@ -49,6 +54,16 @@ export default Controller.extend(bufferedProperty("siteText"), {
     });
   },
 
+  @action
+  dismissOutdated() {
+    this.siteText
```

Comment:
I know it's often not done, but I consider it good practice to return the promise if you have one. Or use async/await